### PR TITLE
add constraints to schema script and update erd

### PR DESCRIPTION
**Pull request by henryjyc**

### CODE REVIEW

File: UtopiaRDS/utopiaAirlineSchema.sql
Comment (kingjon3377) [inline UtopiaRDS/utopiaAirlineSchema.sql]: Throughout, use backticks (`` ` ``), not single quotes (`'`), to quote column names; these constraints are asserting that, for each row, the string constants are not equal, are alphabetically ordered, and are or are not null.

**Review by kingjon3377 — CHANGES_REQUESTED** [review]
Beyond the backtick vs single quote thing, which as we discussed should make these constraints actually work, I wish you had saved the new ERD to the same filename as the old instead of saving to a new name and removing the old. But I won't make that block merging this (you'd have to amend that commit to fix it properly); once you fix the quoting, I'll approve this.

File: UtopiaRDS/utopiaAirlineSchema.sql
Comment (henryjyc) [reply UtopiaRDS/utopiaAirlineSchema.sql]: Good catch. They are changed in the newest pr

**Review by kingjon3377 — APPROVED** [review]
Looks good to me (without testing it locally myself).

File: UtopiaRDS/utopiaAirlineSchema.sql
Comment (STOzaki) [inline UtopiaRDS/utopiaAirlineSchema.sql:24]: Do you really want to have the constraint name contain special characters?

**Review by STOzaki — APPROVED** [review]
Looks good, and thank you for doing this! I do not think that suggesting I have made are critical to keep this PR from getting merged. It is just an observation

File: UtopiaRDS/utopiaAirlineSchema.sql
Comment (kingjon3377) [reply UtopiaRDS/utopiaAirlineSchema.sql:24]: I would agree in general, but this is less important for constraint names than for field names, since our code will never refer to the constraint by name.

Comment (STOzaki) [reply UtopiaRDS/utopiaAirlineSchema.sql]: I agree. And, that is why I made it a suggestion more than anything else.

**Pull request merged**
